### ISSUE TITLE
Use materialized views in kooste schema

### ISSUE THREAD

**Issue by Joonalai** [GispoCoding/tarmo#13]
Use materialized views in kooste schema to make it easier to update rows.

**Comment by SannaJokela** [comment]
@pavetsu14 do we need original LIPAS tables?

**Comment by pavetsu14** [comment]
@SannaJokela Not necessarily. But by storing the original tables in the lipas schema as they are served in Lipas system creates additional knowledge. E.g. if we have an object belonging to luistelukenttä class in the lipas_pisteet table in the kooste schema, we do not really know what kind of attribute data there is available from this kind of objects in Lipas. This follows from the fact that different kind of attribute data gets acquired from sport places with different type codes and in the kooste schema, we only have taken an union of all possible attribute data collected from any of the sport places' type codes we are interested in. So if luistelukenttä object has null on toilet-field, without lipas schema, we have no way of knowing whether this information is missing just from this particular object or whether it is not possible to store for objects belonging to luistelukenttä class at all.

**Comment by SannaJokela** [comment]
@pavetsu14 is this still an issue? Can you close this or assign to Riku.

**Comment by pavetsu14** [comment]
@SannaJokela This might be an issue, depends on whether performance problems emerge. Would be an improvement, but probably not a necessary change
